client/main.html

```diff
@@ -12,17 +12,4 @@
       ga('send', 'pageview');
 
     </script>
-    <div class="container">
```

Comment:
Iron router handles the active view by appending it to the `<body>` tag. It is currently set to display the `home` template at `/` route